src/wp-admin/credits.php

```diff
@@ -135,3 +135,17 @@
 __( 'Release Deputy' );
 __( 'Core Developer' );
 __( 'External Libraries' );
+__( 'Release Coordinator' );
```

Comment:
After a discussion on the thread, it was decided to go with `Release Coordination` as Coordinator is gendered in many languages that use gendered nouns, whereas coordination is not.

```suggestion
__( 'Release Coordination' );
```

Reply:
Thanks for the suggestion @peterwilsoncc. Implemented the changes in the latest commit 🚀.